src/scenes/Partners/Edit/EditPartner.tsx

```diff
@@ -190,6 +191,7 @@ export const EditPartner = ({
                 input: {
                   partner: {
                     ...rest,
+                    address: address?.trim().length ? address : null,
```

Comment:
I think with https://github.com/SeedCompany/cord-api-v3/pull/2469 the field will be cleared now without this. Can you try?

Reply:
I tried yesterday but I think it's still needed. Let me double-check.

Reply:
Ok, just tried and to be able to clear the partner address it's needed to submit a blank space `' '` or more. That doesn't feel very intuitive to me.